packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -71,6 +71,10 @@ export enum EmailEventStatusEnum {
   BOUNCED = 'bounced',
   DROPPED = 'dropped',
   CLICKED = 'clicked',
+  BLOCKED = 'blocked',
+  SPAM = 'spam',
+  UNSUB = 'unsub',
+  SENT = 'sent',
```

Comment:
I think that sent and delievered are the same logical thing. @davidsoderberg @abhilipsasahoo03 what do you think?

Reply:
Well, SENT means the message is on its way to the recipient and DELIVERED means the message has been delivered to the recipient's cell phone and is waiting for them to open it. I don't think they are same in this context either?